src/store/api/userApi/updateUser.ts

```diff
@@ -0,0 +1,40 @@
+import { getToken } from '@utils/tokenStorage';
+import { AuthState } from './types/authTypes';
+
+const apiURL = import.meta.env.VITE_API_URL;
+
+export const updateUser = async (
+  name: string,
+  address: string,
+  phone: string,
+  set: (state: Partial<AuthState>) => void,
```

Comment:
Why did you remove the `_` before the state? Is there any issue with that? I am just telling you because of the annoying warnings that we have when running lint.